src/app/tests/suites/commands/system/scripts/Reboot.py

```diff
@@ -23,5 +23,8 @@
 if sys.platform == 'linux':
     IP = '10.10.10.5'
 
+# Remove the script name
+app = sys.argv.pop(0)
+
 with xmlrpc.client.ServerProxy('http://' + IP + ':' + str(PORT) + '/', allow_none=True) as proxy:
-    proxy.reboot('default', sys.argv[1])
+    proxy.reboot('default', sys.argv)
```

Comment:
Ah, is this why we want to pass a flat list to reboot/start?

In that case, maybe we should document on those methods what the expected input is (instead of documenting it somewhere on the helper function they call) and documenting _why_ that is the input.

This also seems to assume that everything in `argv[1:]` will be key-value pairs.

And why are we doing the `pop` thing, instead of `argv[1:]`?

Reply:
> Ah, is this why we want to pass a flat list to reboot/start?

Yeah, I guess technically I could create the dictionary here but I thought maybe it be better that is done from the direct caller of test_definition.py `start` (which is accessories.py) which is why I created `__createCommandLineOptions` there.

> In that case, maybe we should document on those methods what the expected input is (instead of documenting it somewhere on the helper function they call) and documenting why that is the input.

I'll update documentation

> And why are we doing the pop thing, instead of argv[1:]?

Because I'm not great with python :) I'll change to `argv[1:]`